### ISSUE TITLE
phive fails to install phpbench

### ISSUE THREAD

**Issue by jakzal** [jakzal/toolbox#385]
https://github.com/jakzal/toolbox/runs/2339677014?check_suite_focus=true#step:5:350

```
Phive 0.14.5 - Copyright (C) 2015-2021 by Arne Blankerts, Sebastian Heuer and Contributors
Downloading https://github.com/phpbench/phpbench/releases/download/1.0.0-beta1/phpbench.phar
Downloading https://github.com/phpbench/phpbench/releases/download/1.0.0-beta1/phpbench.phar.asc
Downloading key 15E1F8E2B149E6F5
Trying to connect to keys.openpgp.org (37.218.245.50)
Downloading https://keys.openpgp.org/pks/lookup?op=get&options=mr&search=0x15E1F8E2B149E6F5
Successfully downloaded key.
Warning:  Parsing key data failed with error code 0: No UIDs in key found
Trying to connect to keyserver.ubuntu.com (162.213.33.9)
Trying to connect to keyserver.ubuntu.com (162.213.33.8)
Warning:  Failed with error code 404: Key not found on keyserver
Trying to connect to hkps.pool.sks-keyservers.net (209.244.105.201)
Warning:  Failed with error code 404: Key not found on keyserver
Error:    PublicKey 15E1F8E2B149E6F5 not found on key servers
```

/cc @mihai-stancu @dantleech

**Comment by dantleech** [comment]
i thought i published the key -- but I also thought that PHIVE would not pick up the beta version? https://github.com/phar-io/phive/issues/154 but I'll publish the key shortly anyway.

**Comment by dantleech** [comment]
I've published _and verified_ the key now, but `phive install phpbench` still installs the `0.16.9` version (the last release without a stability flag and an `.asc`)

**Comment by jakzal** [comment]
Thanks @dantleech 🍺 

Fixed in 4da356378a77547acf2e3d95c8c98badf4a5420b
